### DISCUSSION TITLE
Can't seem to make jotai work in CodeSandbox

### DISCUSSION THREAD

**Discussion by aade-sh** [pmndrs/jotai#718] [Q&A] [answered]
I was just trying out jotai on CSB and for some reason it's not able to find the package. This doesn't happen with v6.0.1 used in the demo mentioned in the README but using the latest version throws this error. Any help would be appreciated.

Here's the link to the CSB: [link](https://codesandbox.io/s/kind-payne-i2eb3?file=/src/App.tsx)

**Comment by dai-shi** [comment, marked as answer]
Oh no! I wasn't aware the error in CSB. Thanks for reporting. I will investigate it. Meanwhile, v1.3.3 is the working version.

**Reply by dai-shi** [reply]
I'm pretty sure it worked at some point. Does CSB change something lately? My guess is that they now read "exports" but don't understand wildcards.
cc @barelyhuman

**Reply by barelyhuman** [reply]
@a2chan 
workaround for you if you are blocked because of this for now: https://codesandbox.io/s/relaxed-dubinsky-gqffc?file=/src/App.tsx

@dai-shi CSB has it's own resolution logic that I'm going through right now and while it's confirmed they don't support relative exports, I'm looking for the line of code that needs to be changed there so I can either open an issue or raise a PR for the same for them, since this is to be generalised as node supports it.

**Reply by dai-shi** [reply]
Do you know if CSB changed something recently? I think it worked when I created the PR #704.
Do you think it is because of `*` in package.json exports, because v1.3.3 works ok? Or something else?

**Reply by barelyhuman** [reply]
1.3.3 works well and that's the only major change effecting exports in between 1.3.3 - 1.3.4 , I went through their code and the packager doesn't really have any code with regards to resolving relative subpath patterns. I'm raising a feature request and we'll get our answer or they'll help raise a PR for the same.

We can rollback if this comes up more than once

**Reply by dai-shi** [reply]
https://github.com/codesandbox/codesandbox-client/issues/6147
@barelyhuman filed an issue there, and it turns out it's recently changed.
Looks like it's reverted and now it's working fine with v1.3.5.
